### PR TITLE
Error string should not be capitalized or end with the punctuation mark

### PR DESCRIPTION
**Pull request by CatalinStratu**
Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context. That is, use fmt.Errorf("something bad") not fmt.Errorf("Something bad"), so that log.Printf("Reading %s: %v", filename, err) formats without a spurious capital letter mid-message. This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.

Source: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

### CODE REVIEW

**Comment by CatalinStratu** [comment]
Hi, @lumjjb please review.

**Comment by CatalinStratu** [comment]
@swinslow and @RishabhBhatnagar, what do you think about this PR?

**Review by RishabhBhatnagar — CHANGES_REQUESTED** [review]

File: rdfloader/parser2v2/license_utils_test.go
Comment (CatalinStratu) [reply rdfloader/parser2v2/license_utils_test.go]: @RishabhBhatnagar can you look again?

File: rdfloader/parser2v2/license_utils.go
Comment (RishabhBhatnagar) [inline rdfloader/parser2v2/license_utils.go:83]: avoid adding Function names before the function unless the following statement captures the intent of the function.

File: rdfloader/parser2v2/license_utils.go
Comment (RishabhBhatnagar) [inline rdfloader/parser2v2/license_utils.go:88]: Same comment here and in the other files as well.

**Review by RishabhBhatnagar — CHANGES_REQUESTED** [review]

**Comment by RishabhBhatnagar** [comment]
This is kind of a cosmetic change. I haven't kept track of other PRs. 
I'm afraid that it might cause merge conflicts for them.

**Comment by lumjjb** [comment]
Hi @CatalinStratu ! Yes you are right, this will likely merge conflict with the PR that @kzantow is working on: https://github.com/spdx/tools-golang/pull/172, which involves a major refactor, can we put this on hold until that PR gets in and apply the changes once that gets merged? Which should be before the end of the month 🤞

**Comment by kzantow** [comment]
Hi @CatalinStratu -- apologies for the large number of conflicts! There has been a significant refactoring to make this library easier for consumers; are you able to adjust this PR for the changes?

**Comment by CatalinStratu** [comment]
Hi, @kzantow will adapt this PR for the new changes.

**Comment by lumjjb** [comment]
Closing from inactivity, opened #216 to track since this seems valuable.

**Pull request closed without merging**
